src/animation.jl

```diff
@@ -198,7 +219,13 @@ function _animate(forloop::Expr, args...; callgif = false)
     push!(block.args, :($countersym += 1))
 
     # add a final call to `gif(anim)`?
-    retval = callgif ? :(Plots.gif($animsym)) : animsym
+    retval = if type == :gif
+        :(Plots.gif($animsym))
+    elseif type == :apng
+        :(Plots.apng($animsym))
+    else
+        animsym
+    end
```

Comment:
```suggestion
    retval = if type === :gif
        :(Plots.gif($animsym))
    elseif type === :apng
        :(Plots.apng($animsym))
    else
        animsym
    end
```

I missed that, symbol comparisons should be made with `===`.